<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="火乐乐的电脑" name="Test" tests="19" failures="18" timestamp="10 Dec 2017 09:31:53 GMT" time="41.002" errors="0">
  <testcase name="testPay" time="0.032" classname="test.PayShopping">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at manage.Pay.pay(Pay.java:23)
	at test.PayShopping.testPay(PayShopping.java:14)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- testPay -->
  <testcase name="testDeleteAddress" time="0.019" classname="test.Address">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at manage.DeleteAddress.delete(DeleteAddress.java:13)
	at test.Address.testDeleteAddress(Address.java:22)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- testDeleteAddress -->
  <testcase name="testAddShopping" time="0.021" classname="test.Shopping">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at manage.AddShopping.addgoods(AddShopping.java:12)
	at test.Shopping.testAddShopping(Shopping.java:17)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- testAddShopping -->
  <testcase name="testDeleteShopping" time="0.005" classname="test.Shopping">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at manage.DeleteShopping.delete(DeleteShopping.java:12)
	at test.Shopping.testDeleteShopping(Shopping.java:22)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- testDeleteShopping -->
  <testcase name="testAddAddress" time="3.02" classname="test.Address">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at manage.AddAddress.input(AddAddress.java:15)
	at test.Address.testAddAddress(Address.java:17)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- testAddAddress -->
  <testcase name="testAddMessage" time="3.02" classname="test.Massage">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at manage.AddMassage.input(AddMassage.java:13)
	at test.Massage.testAddMessage(Massage.java:17)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- testAddMessage -->
  <testcase name="testAddMessage" time="3.016" classname="test.Massage">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at manage.AddMassage.input(AddMassage.java:13)
	at test.Massage.testAddMessage(Massage.java:17)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- testAddMessage -->
  <testcase name="testAddAddress" time="3.016" classname="test.Address">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at manage.AddAddress.input(AddAddress.java:15)
	at test.Address.testAddAddress(Address.java:17)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- testAddAddress -->
  <testcase name="testAddAddress" time="3.014" classname="test.Address">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at manage.AddAddress.input(AddAddress.java:15)
	at test.Address.testAddAddress(Address.java:17)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- testAddAddress -->
  <testcase name="testAddMessage" time="3.071" classname="test.Massage">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at manage.AddMassage.input(AddMassage.java:13)
	at test.Massage.testAddMessage(Massage.java:17)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- testAddMessage -->
  <testcase name="testAddAddress" time="3.026" classname="test.Address">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at manage.AddAddress.input(AddAddress.java:15)
	at test.Address.testAddAddress(Address.java:17)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- testAddAddress -->
  <testcase name="testAddMessage" time="3.002" classname="test.Massage">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at manage.AddMassage.input(AddMassage.java:13)
	at test.Massage.testAddMessage(Massage.java:17)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- testAddMessage -->
  <testcase name="testLogin" time="13.566" classname="test.Login"/>
  <testcase name="testAddAddress" time="3.005" classname="test.Address">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at manage.AddAddress.input(AddAddress.java:15)
	at test.Address.testAddAddress(Address.java:17)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- testAddAddress -->
  <testcase name="testAddMessage" time="3.013" classname="test.Massage">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at manage.AddMassage.input(AddMassage.java:13)
	at test.Massage.testAddMessage(Massage.java:17)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- testAddMessage -->
  <testcase name="testAddMessage" time="3.013" classname="test.Massage">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at manage.AddMassage.input(AddMassage.java:13)
	at test.Massage.testAddMessage(Massage.java:17)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- testAddMessage -->
  <testcase name="testAddMessage" time="3.013" classname="test.Massage">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at manage.AddMassage.input(AddMassage.java:13)
	at test.Massage.testAddMessage(Massage.java:17)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- testAddMessage -->
  <testcase name="testAddMessage" time="3.013" classname="test.Massage">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at manage.AddMassage.input(AddMassage.java:13)
	at test.Massage.testAddMessage(Massage.java:17)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- testAddMessage -->
  <testcase name="testAddMessage" time="3.014" classname="test.Massage">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at manage.AddMassage.input(AddMassage.java:13)
	at test.Massage.testAddMessage(Massage.java:17)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- testAddMessage -->
</testsuite> <!-- Test -->
